Gaijinizer patch/Scripts.txt

```diff
@@ -2957,46 +2952,46 @@ You have successfully completed this game!
 次の世界の『僕』に、よろしくね。
  =================================================="
 > CONTEXT: Scripts/テキスト出力/126:12
-"Subject: To 'interventor'
-sender【___】
+"Sender: Unknown
+Subject: To the 'Intervener'
 
- 《Text》==========================================
-...... Is it properly received?
- Invents a machine that sends messages directly to Dr. Kurosawa.
-I'm skeptical and I am using it ...
-If this has not arrived, it's my grand soliloquy ...
-
-You know my name, right?
- You gave me a little through Meira when I was young ...
-You made me escape from that city.
- I can not thank you.
-
- I have no intention to blame you.
-That is what Reina wanted.
-And as she says...I couldn't close the trigger.
-Well, if the decision was a little late,
- I and Reina were also included in Samael.
- Her resolution was about to be wasted.
-
-But if you say that there is no regret...it will be a lie.
- Of course, I intended to act so as not to regret myself.
-But if there was a way to save her ...
-
-One thing I would like to ask.
-If you roll around the world, you have another end to this end
-If you have the courage to see...I want you to save her again this time.
-
- I of 'this world' can not see the happy ending.
-But you just find out the possibility
-For me, it should be a big power ......
-
-Unfortunately I can not help but do not worry.
-'My girls' in my room will help you
-The power of Kizuna that you have spun up will surely not be wasted.
-
-Well then...I think that you are goodbye already ...
-Thank you to the next world 's' I'.
- =================================================="#MTLed
+
+ 《Body》==========================================
+...Is this getting through?
+I'm using this device Professor Yui invented to send a message
+directly to the 'Intervener'. Honestly I'm a little skeptical it'll
+work. If this isn't going anywhere then I'm just making a grand speech
+to myself.
```

Comment:
A bit of localization/refinement to make this smoother:

>_**Honestly, I'm a little skeptical that
it'll work... if it isn't going anywhere then I'm just going to be
making a grand monologue to myself.**_